### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kvaps**
- Add basic project information
- Move documentation to cozystack.io

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,548 +18,53 @@ You can use Cozystack to build your own cloud or to provide a cost-effective dev
 
 ## Use-Cases
 
-### As a backend for a public cloud
+* [**Using Cozystack to build public cloud**](https://cozystack.io/docs/use-cases/public-cloud/)  
+You can use Cozystack as backend for a public cloud
 
-Cozystack positions itself as a kind of framework for building public clouds. The key word here is framework. In this case, it's important to understand that Cozystack is made for cloud providers, not for end users.
+* [**Using Cozystack to build private cloud**](https://cozystack.io/docs/use-cases/private-cloud/)  
+You can use Cozystack as platform to build a private cloud powered by Infrastructure-as-Code
```
Comment (gecube) [inline README.md]: "powered by Infrastructure-as-Code approach"

File: README.md [hunk 2]
```diff
@@ -18,548 +18,53 @@ You can use Cozystack to build your own cloud or to provide a cost-effective dev
...
-```bash
-kubectl get secret -n tenant-root tenant-root -o go-template='{{ printf "%s\n" (index .data "token" | base64decode) }}'
-```
+## Documentation
 
-Enable port forward to cozy-dashboard:
-```bash
-kubectl port-forward -n cozy-dashboard svc/dashboard 8080:80
-```
+The documentation located on offical [cozystack.io](cozystack.io) website.
```
Comment (gecube) [inline README.md]: located -> "is located"

Comment (gecube) [reply README.md]: "offical" -> "official"

**Comment by kvaps** [comment]
@gecube thanks for corrections, PR rebased

**Review by gecube — APPROVED** [review]

**Pull request merged**